README.md

```diff
@@ -672,6 +672,18 @@ Circus Train can be extended in various ways. This is an advanced feature that t
 * `LocationManagers` look after the source and replica locations and are an ideal point to implement both snapshot isolation and retired data clean-up.
 * `CompositeCopierFactory` allows the provision of multiple copiers for the same table in order to add functionality to the copy process, e.g. introduce compaction, copy to multiple destinations, etc.
 
+### External Extensions
+
+#### Circus Train DataSqueeze Copier
+This extension implements new Copiers for Circus Train which integrate DataSqueeze.
```

Comment:
Either make this:
```
This extension implements new Copiers for Circus Train which integrate DataSqueeze:
```
note the `:`.

*Or* make the word `DataSqueeze` linkable like this and remove the separate URL:
```
This extension implements new Copiers for Circus Train which integrate [DataSqueeze](https://github.com/ExpediaInceCommercePlatform/circus-train-datasqueeze).
```

```

Reply:
could you please check this again,  I have added that.